app/assets/stylesheets/spotlight/_blacklight_configuration.scss

```diff
@@ -30,14 +30,6 @@
     padding-top: 0;
     padding-bottom: 0;
 
-    label {
-      margin-left: $spacer;
-      margin-right: $spacer;
-      padding-left: 0px;
-      padding-bottom: $table-cell-padding;
-      padding-top: 1 + $table-cell-padding;
-      display: block;
-    }
```

Comment:
`$table-cell-padding` is not available in bootstrap 5. This particular label is `visually-hidden`/`sr-only` and didn't seem to impact the display at all so I removed the styling.